molgenis-metadata-manager/src/main/frontend/src/store/utils/flow.types.js

```diff
@@ -0,0 +1,98 @@
+// @flow
+export type Alert = {
+  'message': ?string,
```

Comment:
You use the same mutation to set everything to null..
or is there another way to indicate it is allowed to be null?